src/thread.ts

```diff
@@ -0,0 +1,14 @@
+export function createThreadBody(
+  threadName?: string,
+  threadKey?: string
+): Record<string, string> {
+  if (threadName) {
+    return { name: threadName }
+  }
+  if (threadKey) {
+    return {
+      threadKey
+    }
+  }
+  return {}
```

Comment:
Since this should never happen, can you please throw an error here?
```suggestion
throw new Error("Either threadName or threadKey must be provided");
```